### ISSUE TITLE
stick-to-page doesn't work

### ISSUE THREAD

**Issue by vfl-borussia-mg** [VadimDez/ng2-pdf-viewer#244]
##### Bug Report or Feature Request (mark with an `x`)
```
- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [X] Bug report -> please search issues before submitting
- [ ] Feature request
- [ ] Documentation issue or request
```
Setting a new page in the stick-to-page mode (in version 3.0.5) doesn't work for me. show-all and stick-to-page are set to "true" as you can see in my code ...

```
import { Component, OnInit } from '@angular/core';

@Component({
  selector: 'app-root',
  template: `
  <div>
      <label>PDF src</label>
      <input type="text" placeholder="PDF src" [(ngModel)]="pdfSrc">
      <button (click)="switchShowAll()">showAll</button>
      <button (click)="switchSticky()">sticky</button>
      <div *ngIf="stickToPage">
        <button (click)="setPage(-1)">prev</button>
        <button (click)="setPage(1)">next</button>
      </div>
  </div>
  <pdf-viewer [src]="pdfSrc" 
              [(page)]="page"
              [show-all]="showAll"
              [stick-to-page]="stickToPage"
              [render-text]="false"
              (after-load-complete)="onAfterLoad($event)"
              style="display: block;">
  </pdf-viewer>
  `
})
export class AppComponent implements OnInit {
  pdfSrc: string = '../assets/pdf-test.pdf';
  page: number = 1;
  stickToPage: boolean = false;
  showAll: boolean = true;

  ngOnInit(){   
  }

  onAfterLoad(event: any){
  }

  switchSticky(){
    this.stickToPage = !this.stickToPage;
  }

  switchShowAll(){
    this.showAll = !this.showAll;
  }

  setPage(num: number){
    this.page += num;
    console.log(this.page);
  }
}
```
... and I use angular-cli 1.6.3 as well.

```
Angular CLI: 1.6.3
Node: 9.3.0
OS: win32 x64
Angular: 5.1.3
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

@angular/cli: 1.6.3
@angular-devkit/build-optimizer: 0.0.36
@angular-devkit/core: 0.0.22
@angular-devkit/schematics: 0.0.42
@ngtools/json-schema: 1.1.0
@ngtools/webpack: 1.9.3
@schematics/angular: 0.1.11
@schematics/schematics: 0.0.11
typescript: 2.4.2
webpack: 3.10.0
```
Regards & thanks for investigation

**Comment by vfl-borussia-mg** [comment]
I played around with your example from following blog post:

https://medium.com/@vadimdez/render-pdf-in-angular-4-927e31da9c76

I just modified the app.component.html in the way below:

```
<div *ngIf="isLoaded" style="text-align: center;">
  <button (click)="prevPage()" [disabled]="page === 1">Prev</button>
  <span>{{ page }} / {{ totalPages }}</span>
  <button (click)="nextPage()" [disabled]="page === totalPages">Next</button>
</div>

<pdf-viewer src="https://vadimdez.github.io/ng2-pdf-viewer/pdf-test.pdf"
            [show-all]="true"
            [stick-to-page]="true"
            [page]="page"
            (after-load-complete)="afterLoadComplete($event)"
></pdf-viewer>
```
As you can see, I changed show-all from "false" to "true" and added stick-to-page property an set them to true. I expected that the pdf view jump to the requestet page, but it didn't.

**Comment by michaelBaginski** [comment]
I've played with it, too. My experience is, that the jump to the correct page (when show-all is set to true) does not work, when the requested page is set initially in the constructor or in the init-Method, even not in the afterload-complete function (if you do that, the component behaves rather strange) - it worked, when I wait a bit longer, e.g. putting  

setTimeout(() => {
            this.page = 28;
        }, 500); 

in the afterload-complete method  but that's kinda dirty.. ;-)

**Comment by VadimDez** [comment]
@vfl-borussia-mg I think I could reproduce your problem and what i can tell so far that if you have `<!doctype html>` in your `index.html` - that's the problem. Try to comment it out and see what happens.

**Comment by vfl-borussia-mg** [comment]
Yeah, you are right, this helps. Thanks for the workaround. 
But, without doc type validation i maybe get other problems.
Will this issue fixed in one of a further releases?

**Comment by VadimDez** [comment]
@vfl-borussia-mg Yes, this should be definitely fixed in the future!

**Comment by SIlver--** [comment]
Was this ever fixed? I noticed in the demo it still has `<!doctype html>` disabled.

**Comment by elandwz** [comment]
Will fix this <!doctype html> disabled issue, it really will cause problems.

**Comment by VadimDez** [comment]
Fixed in `6.0.0` See https://github.com/VadimDez/ng2-pdf-viewer/blob/master/CHANGELOG.md#600
